src/utils/toDate.ts

```diff
@@ -0,0 +1,9 @@
+import BigNumber from 'bignumber.js';
+import { toBigNumber } from './toBigNumber';
+
+export function toDate(timestamp: string | number | BigNumber) {
+  const result = toBigNumber(`${timestamp}`)
```

Comment:
toBigNumber() accepts `any` as a parameter. So no need to stringify here. Just `toBigNumber(timestamp)`.